twingate/client.go

```diff
@@ -135,33 +158,30 @@ func NewClient(network, apiToken, url string) *Client {
 	client := Client{
 		HTTPClient:       httpClient,
 		ServerURL:        serverURL,
-		GraphqlServerURL: fmt.Sprintf("%s/api/graphql/", serverURL),
-		APIServerURL:     fmt.Sprintf("%s/api/v1", serverURL),
+		GraphqlServerURL: graphqlServerURL,
+		APIServerURL:     apiServerURL,
 		APIToken:         apiToken,
+		GraphqlClient:    graphql.NewClient(graphqlServerURL, &c),
 	}
-	log.Printf("[INFO] Using Server URL %s", client.ServerURL)
+
+	log.Printf("[INFO] Using Server URL %s", graphqlServerURL)
 
 	return &client
 }
 
+type pingQuery struct {
```

Comment:
I dont think we need that ping query any more , i just used for our test client , so lets remove it

Reply:
providerConfigure() is using ping()
https://github.com/Twingate/terraform-provider-twingate/blob/a6035e196ff29c2817f0c0807a1d8abcc487f82b/twingate/provider.go#L63

Reply:
Remove the ping query and its usages